scripts/devbase.sh

```diff
@@ -6,25 +6,42 @@ DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
 libDir="$DIR/../.bootstrap"
 lockfile="$DIR/../stencil.lock"
 serviceYaml="$DIR/../service.yaml"
+gojqVersion="v0.12.14"
 
 # get_absolute_path returns the absolute path of a file
 get_absolute_path() {
   python="$(command -v python3 || command -v python)"
   "$python" -c "import os,sys; print(os.path.realpath(sys.argv[1]))" "$1"
 }
 
-# get_field_from_yaml reads a field from a yaml file using either go-yq or python-yq
+# get_field_from_yaml reads a field from a yaml file via a JIT-downloaded gojq.
 get_field_from_yaml() {
-  field="$1"
-  file="$2"
+  local field="$1"
+  local file="$2"
 
-  if [[ "$(yq e '.a' '-' <<<'{"a": "true"}' 2>&1)" == "true" ]]; then
-    # using golang version
-    yq e "$field" "$file"
-  else
-    # probably using python version
-    yq -r "$field" <"$file"
+  local gjDir
```

Comment:
**thought:** Probably best to compose this logic into a separate function and call out to it? Perhaps run it whenever this library is imported (sourced)?